tests/test_workers/test_tasks/test_build.py

```diff
@@ -1267,3 +1306,19 @@ def test_adjust_csv_annotations_no_customizations(mock_yaml_dump, tmpdir):
     build._adjust_csv_annotations(operator_manifest.files, 'amqp-streams', 'mos-eisley')
 
     mock_yaml_dump.assert_not_called()
+
+
+def test_add_ocp_label_to_index(tmpdir):
+    operator_dir = tmpdir.mkdir('operator')
+    operator_dir.join('Dockerfile').write(
+        'https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8-minimal'
+    )
+
+    build._add_ocp_label_to_index('v4.5', operator_dir, 'Dockerfile')
+
+    assert (
+        operator_dir.join('Dockerfile').read_text('utf-8')
+        == """https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8-minimal
+LABEL com.redhat.index.delivery.version="v4.5"
+"""
+    )
```

Comment:
This is easier to read in my opinion.
```suggestion
    expected = textwrap.dedent(
        '''\
        https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8-minimal
        LABEL com.redhat.index.delivery.version="v4.5
        ''''
    )
    assert operator_dir.join('Dockerfile').read_text('utf-8') == expected
```

Reply:
+1 did not know about textwrap thanks!